standard/date_ext.lua

```diff
@@ -21,6 +21,9 @@ DateExt.minTimestamp = -62167219200
 -- 9999-12-31 23:59:59
 DateExt.maxTimestamp = 253402300799
 
+-- 1970-01-01 00:00:00
+DateExt.timestampZero = 0
```

Comment:
Maybe `DateExt.zero` or `DateExt.epoch`?

Reply:
I prefer keeping it this way, although I agree it doesn't _feel_ ideal

Reply:
Got a few other possible names, any of them that speaks to you?
* `DateExt.timestampEpoch`
* `DateExt.epochTimestamp`
* `DateExt.epochZero`
* `DateExt.epochStart`
* `DateExt.startEpoch`

Reply:
Let's take `epochZero`